detections/endpoint/windows_anonymous_pipe_activity.yml

```
@@ -0,0 +1,62 @@
+name: Windows Anonymous Pipe Activity
+id: ee301e1e-cd81-4011-a911-e5f049b9e3d5
+version: 1
+date: '2025-02-11'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
```

Since you found FPs in `Program Files` this means that other binaries found in `Program Files (x86)`, `ProgramData` or even `AppData` can showcase the same behavior. Hence it cannot be an anomaly on its own.

```suggestion
type: Hunting
```